### ISSUE TITLE
Advise results: which are more useful than the other

### ISSUE THREAD

**Issue by TomasTomecek** [thoth-station/support#259]
**Is your feature request related to a problem? Please describe.**

Just going through the latest results in my console for https://thoth-station.ninja/search/adviser-220719131518-f9cde332e97c5073/summary and giving you feedback for the output. I separate every point with a \<hr\>. Let's go!

```
Multiple imports for...
```
There are too many lines on the output, I have no control over the code inside our dependencies.

---

```
No version range specifier...
```
Could be a single line with multiple dependencies, doesn't need to be a single line for every dep. I also understand this is important for people that install directly from PyPI.

---

A lot of the `INFO` lines could even be suppressed: `No $tool used`

---

```
Runtime environment used is no longer supported, it is recommended to switch to another runtime environment
```
Please write which environment, I don't understand what I should change.

---

```
No CPU model provided...
```
I don't care about this and specifically don't have control over CPU in our CI.

---

```
Skipping including package ('urllib3', '1.25.5', 'https://pypi.org/simple') as a CVE 'PYSEC-2020-149' was found
```
I don't understand if there is anything I should change or if Thoth is just telling me that it doesn't take that CVE into account.

---

Stack report: I like the table in the Web UI with ❌ and ✅: that's easy to read. In the console, there is a lot of text that's hard to parse. Could there e.g. be some score number that would tell the healthiness? So that in console it would only be a single line per project.

---

That's it! Overall the web interface is easier to read for me, especially the stack report. I also appreciate the Compare tab so that I can see what has changed in the meantime. One thing I'd highlight is that it's hard for me to find any of the points actionable - it's good that I know that information but it's not in my control do anything about it. Examples of actionable items:

* "Dependency $XYZ has low health project status and can be replaced with $ASD"

* "You are using setup.py style of Python packaging. Please consider migrating to $QWE"

* "Option $A in setup.cfg is deprecated, please use $B"

**Comment by goern** [comment]
might be related to https://github.com/thoth-station/core/issues/417

**Comment by Gkrumbach07** [comment]
will be fixed by the completion of this epic https://github.com/thoth-station/core/issues/417
